### PR TITLE
Prototype for task creation

### PR DESCRIPTION
**Pull request by ryosuzuki**
Prototype for task creation view. Related issues are #76 and #84.
Trello: https://trello.com/c/LJrLPfiJ

It is still an early prototype though.
345a9f04d1f9f1064fcf37aa3783deaf

### CODE REVIEW

**Comment by neilthemathguy** [comment]
Let's enhance this to address #84

**Comment by Macbull** [comment]
Great ! But according to current Data Model, category is in a Many to Many relationship with project and module !

File: staticfiles/js/tasks/controllers/task.controller.js [hunk 1]
```diff
@@ -1,29 +1,76 @@
...
+      { field: "name", displayName: 'Name', width: 200, pinned: true },
+      { field: "description", displayName: 'Description', width:150 },
+      { field:"keywords",displayName: 'Keywords', width:190 },
+      { field: "module_timeout", displayName:'Hours Remain', width:130 },
+      { field: "repetition", displayName: 'Repetition', width: 100 },
+      { field: "price", displayName: 'Pay', width: 60 },
+    ]
+  };
+
+  var vm = $scope;
```
Comment (nistala) [inline staticfiles/js/tasks/controllers/task.controller.js]: let's have a service to pull these dynamically from the backend?

File: staticfiles/js/tasks/controllers/task.controller.js [hunk 2]
```diff
@@ -1,29 +1,76 @@
...
+    $event.preventDefault();
+    alert(JSON.stringify(vm.task));
+  }
+
+  vm.showStep = function (step) {
+    vm.currentStep = step;
+  };
+
+  vm.datePickerOpen = true;
+  vm.toggleDatePicker = function($event) {
```
Comment (nistala) [inline staticfiles/js/tasks/controllers/task.controller.js]: we can use these date and time pickers http://mgcrea.github.io/angular-strap/#/datepickers

File: staticfiles/templates/task/new.html [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+    </div>
+
+  </form>
+
+</div>
+
+
+<style>
+
+.category {
```
Comment (nistala) [inline staticfiles/templates/task/new.html]: please move styles to a separate file

Comment (ryosuzuki) [inline staticfiles/templates/task/new.html]: Done

File: staticfiles/js/tasks/controllers/task.controller.js (hunk 2 above)
Comment (ryosuzuki) [inline staticfiles/js/tasks/controllers/task.controller.js]: Done

File: staticfiles/js/tasks/controllers/task.controller.js (hunk 1 above)
Comment (ryosuzuki) [inline staticfiles/js/tasks/controllers/task.controller.js]: Create task service.

**Comment by ryosuzuki** [comment]
Resolved all of reviews by @nistala . 
1. Separate CSS file,  
2. Use Datepicker Directive of AngularStrap
3. Create service that get categories from back=end. 

Ready to be merged.

**Comment by dmorina** [comment]
let's keep improving the integrated workflow as we did last week ;), closing this for now

**Pull request closed without merging**